ferret/explainers/explanation_speech/equal_width/gradient_equal_width_explainer.py

```diff
@@ -117,7 +113,7 @@ def compute_explanation(
             importances = []
             a, b = 0, 0  # 50, 20
 
-            duration_s = len(audio_as) / 1000
+            duration_s = len(audio_array) / audio.sample_rate # finds the duration from the array 
```

Comment:
Let's double-check this: the original factor `1000` was converting fro milliseconds to seconds (so `audio_as` was assumed to be an array with lengths equal the duration in milliseconds) while dividing by the sample rate requires that `audio_array` is now as long as the number of samples for a frequency equals to the sample rate (in hz).

Reply:
@emanuele-moscato we should check this one, too